src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -129,7 +129,7 @@ public class ExtensionsRunner {
      */
     private final Injector injector;
 
-    private final SDKNamedXContentRegistry sdkNamedXContentRegistry;
+    private SDKNamedXContentRegistry sdkNamedXContentRegistry;
```

Comment:
Please keep this as `final`.  Also all the existing handlers down at line 138 are now final with a just-merged PR.